packages/augur-ui/src/modules/account/components/open-markets.tsx

```diff
@@ -8,7 +8,7 @@ import { SizeTypes } from "modules/types";
 import Styles from "modules/account/components/open-markets.styles.less";
 
 function filterComp(input: any, market: any) {
-  return market.description.toLowerCase().indexOf(input.toLowerCase()) >= 0;
+  return market.description ? market.description.toLowerCase().indexOf(input.toLowerCase()) >= 0 : true;
```

Comment:
for edge cases when the market doesn't have a description, just include it in the list.